docsets/PyMongo/docset.json

```diff
@@ -1,15 +1,19 @@
 {
     "name": "PyMongo",
-    "version": "3.10.1",
+    "version": "3.12.0",
     "archive": "PyMongo.tgz",
     "author": {
-      "name": "Torin Kwok",
-      "link": "https://github.com/torinkwok"
+      "name": "Julian Torres",
+      "link": "https://github.com/macintacos"
     },
     "aliases": [
         "pymongo"
     ],
     "specific_versions": [
+        {
+            "version": "3.12.0",
+            "archive": "versions/3.12.0/PyMongo.tgz"
```

Comment:
This archive doesn't exist. You added `versions/3.12.0/PyMongo.docset.tgz` instead.